ganttproject/build.gradle

```diff
@@ -79,6 +79,9 @@ jooq {
                     target {
                         packageName = 'biz.ganttproject.storage.db'
                     }
+                    generate {
+                        javaTimeTypes = false
```

Comment:
Can you please add a comment explaining this trick?

Reply:
Added a comment
I'm not sure that the flag is necessary but the date/time conversion seems to be nicer this way as `java.sql.Timestamp` can be constructed using `timeInMillis`. Without it, jooq required using `java.time.LocalDateTime`